nbformat/v4/nbformat.v4.schema.json

```diff
@@ -141,6 +141,15 @@
                 "type": "boolean"
               }
             },
+            "editable": {
+              "description": "Whether the cell is editable.",
+              "type": "boolean"
+            },
+            "deletable": {
+              "description": "Whether the cell is deletable.",
+              "type": "boolean"
+            },
```

Comment:
Each of these new optional boolean fields need to have `"default": true` since the absence of these fields would likely be interpreted as `false`. That or it ends up in ambiguous territory even though we'd probably want to assume these are true in notebooks.

Something like:

```
"editable": {
  "description": "Whether the cell is editable.",
  "type": "boolean",
  "default": true
},
```

for each